openedx_events/data.py

```diff
@@ -34,7 +34,7 @@ class EventsMetadata:
 
     id = attr.ib(type=UUID, init=False)
     event_type = attr.ib(type=str)
-    minorversion = attr.ib(type=int, converter=attr.converters.default_if_none(0))
+    minorversion = attr.ib(type=int, default=None, converter=attr.converters.default_if_none(0))
```

Comment:
@mariajgrimaldi: Since we already had a `default_if_none` converter, and we could have used `EventsMetadata(..., minorversion=None)`, I presume it is acceptable to not require it to be supplied at all? Please confirm that this wasn't because we want to force it as an argument, in which case we should probably go the other way and remove `default_if_none` or something. Let me know what you think of this PR. Thanks!

Reply:
You're right! We don't need it here since we have the converter :)